### PR TITLE
pygamer definition & autobuild

### PR DESCRIPTION
**Pull request by ladyada**
its similar to pybadge, has pin defs for joystick - comes up with filesys and display so good enough to start hacking :)

### CODE REVIEW

File: ports/atmel-samd/boards/pygamer/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+USB_PRODUCT = "PyGamer"
+USB_MANUFACTURER = "Adafruit Industries LLC"
+
+QSPI_FLASH_FILESYSTEM = 1
+EXTERNAL_FLASH_DEVICE_COUNT = 1
+EXTERNAL_FLASH_DEVICES = GD25Q64C
+LONGINT_IMPL = MPZ
+
+# No I2S on SAMD51G
+CIRCUITPY_AUDIOBUSIO = 0
```
Comment (dhalbert) [inline ports/atmel-samd/boards/pygamer/mpconfigboard.mk]: Should we turn this off? It's a SAMD51J. (Comment is misleading.)

Comment (ladyada) [reply ports/atmel-samd/boards/pygamer/mpconfigboard.mk]: oops thats right ill fix - can you do the pybadge?

Comment (ladyada) [reply ports/atmel-samd/boards/pygamer/mpconfigboard.mk]: ok i think done!

**Comment by tannewt** [comment]
Sorry I didn't approve earlier. My radio buttons had gone missing. Thanks @dhalbert for the review.

**Pull request merged**